docs/src/pages/guides/minimizing-bundle-size/minimizing-bundle-size.md

```diff
@@ -226,18 +220,20 @@ It will perform the following diffs:
 +import { Button, TextField } from '@material-ui/core';
 ```
 
-## ECMAScript
+## Available bundles
 
 The package published on npm is **transpiled**, with [Babel](https://github.com/babel/babel), to take into account the [supported platforms](/getting-started/supported-platforms/).
 
-A second version of the components is also published, which
-you can find under the [`/es` folder](https://unpkg.com/@material-ui/core/es/).
-All the non-official syntax is transpiled to the [ECMA-262 standard](https://www.ecma-international.org/publications/standards/Ecma-262.htm), nothing more.
+⚠️ In order to minimize duplication of code in users' bundles, library authors are **strongly discouraged** to import from any of the other bundles.
```

Comment:
Sorry for commenting on a closed PR, but is this recommendation saying for libraries to only import either `/modern` or `/legacy`? Or is it saying to keep imports as they were before, e.g. `@material-ui/core/Button` (which would later be used alongside a Babel transform that repoints at either `/legacy` or `/modern`, when you build an actual app)?

Reply:
Libraries should not deal with from where they import. This should all be up to the bundler. So yes, import only from `@materialui/*/*` in your library.